typescript-template-literals/main.ts

```diff
@@ -0,0 +1,22 @@
+const number: number = 9200;
+const street: string = 'Irvine Center Drive';
+const address: string = `${number} ${street}`;
+
+console.log('LearningFuze:', address);
+
+const firstName: string = 'Uzair';
+const lastName: string = 'Ashraf';
+const age: number = 25;
+
+const bio: string = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+console.log('Bio:', bio);
+
+const fruit: string = 'apple';
+const item: string = 'pen';
+
+const applePen: string = `${fruit} ${item}`;
```

Comment:
Technically there should be no space here!

Reply:
I'll make the correction.

Reply:
![Screenshot 2024-12-10 at 9 48 30 AM](https://github.com/user-attachments/assets/d333f06b-15fc-4de0-a318-572f8fbc5c9f)